.github/workflows/install-test-build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           cache: npm
       - name: 'NPM: Add Config and Authorization'
         run: |
-          rm .npmrc
+          rm -f .npmrc
```

Comment:
Why do you need to remove .npmrc when the echo command below overrides its contents?